public/app/features/alerting/unified/RuleEditorCloudRules.test.tsx

```diff
@@ -45,7 +45,7 @@ describe('RuleEditor cloud', () => {
   it('can create a new cloud alert', async () => {
     const { user } = renderRuleEditor();
 
-    const removeExpressionsButtons = screen.getAllByLabelText('Remove expression');
+    const removeExpressionsButtons = screen.getAllByLabelText(/Remove expression/);
```

Comment:
I updated the `Remove expression` to reference the node that it relates to. This is "obvious" from the UI context, but making it clearer from an a11y point of view is an improvement, as we otherwise have multiple elements on the screen that say the same thing ("Remove expression")

This means this should be a regex check rather than string